readthedocs/projects/views/mixins.py

```diff
@@ -44,3 +47,46 @@ def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
         context[self.project_context_object_name] = self.get_project()
         return context
+
+
+class ProjectImportMixin:
+
+    """Helpers to import a Project."""
+
+    def import_project(self, project, tags, request):
+        """
+        Import a Project into Read the Docs.
```

Comment:
As you noted in the PR comment above, we should note what work it assumes has already been done in the code comment as well.